src/app/clusters/service-area-server/service-area-cluster-objects.h

```diff
@@ -38,8 +38,9 @@ struct LocationStructureWrapper : public chip::app::Clusters::ServiceArea::Struc
 {
     LocationStructureWrapper()
     {
-        Set(0, 0, CharSpan(), DataModel::Nullable<int16_t>(), DataModel::Nullable<AreaTypeTag>(),
-            DataModel::Nullable<LandmarkTag>(), DataModel::Nullable<PositionTag>(), DataModel::Nullable<FloorSurfaceTag>());
+        Set(0, 0, CharSpan(), DataModel::Nullable<int16_t>(), DataModel::Nullable<Globals::AreaTypeTag>(),
+            DataModel::Nullable<Globals::LandmarkTag>(), DataModel::Nullable<Globals::PositionTag>(),
+            DataModel::Nullable<Globals::FloorSurfaceTag>());
```

Comment:
While you are here, just using `NullNullable` here would be shorter, but maybe we want to make it clear which things we are leaving null?

Reply:
I'm going to move this note the [PR that's adding a builder](https://github.com/project-chip/connectedhomeip/pull/34683/files) since there are a lot of similar changes being made there.

Reply:
Added the note [here](https://github.com/project-chip/connectedhomeip/pull/34683#discussion_r1699742228).